### PR TITLE
#444 - GetUserSlice tested and added to ArtipieApi

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #444 - created test for `GetUserSlice` and added corresponding endpoint to `ArtipieApi`.

### CODE REVIEW

File: src/main/java/com/artipie/api/ArtipieApi.java [hunk 1]
```diff
@@ -58,7 +59,7 @@
  * @since 0.6
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
-@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+@SuppressWarnings({"PMD.AvoidDuplicateLiterals", "PMD.ExcessiveMethodLength"})
```
Comment (olegmoz) [inline src/main/java/com/artipie/api/ArtipieApi.java:62]: @olenagerasimova I'd suggest to add a puzzle about some decomposition of constructor, it's hard to read

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, it all looks good, have one suggestion though

**Comment by olenagerasimova** [comment]
@olegmoz thanks, puzzle added

**Review by olegmoz — APPROVED** [review]

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#462` is not assigned, can't get performer

**Pull request merged**